### PR TITLE
SOOP CO2 NRT: process missing variable

### PR DESCRIPTION
**Pull request by bpasquer**
remove deprecated config(aurora australis)

edit variable name and mapping

update netcdf template: new and upd variables
update Netcdf generator script to amend parameter+remove ref to aurora update unittests: use new RT file+remove ref to RT aurora

add new unittest file and remove obsolete

### CODE REVIEW

File: aodndata/soop/soop_co2_nrt_nc_generator.py [hunk 1]
```diff
@@ -234,27 +221,22 @@ def check_parameters(dataf, vessel_code, input_param, src_file):
...
+    missing_param = []
+    if not all(param in input_param for param in INPUT_RT_PARAMETERS):
+        for required_param in INPUT_RT_PARAMETERS:
             if required_param not in input_param:
-                missing_param = missing_param.append(required_param)
-                raise InvalidFileContentError(
-                    "Missing parameter(s) '{missing_param}' in file '{src_file}'.Aborting".format(
-                        missing_param=missing_param, src_file=src_file))
+                missing_param.append(required_param)
     else:  # required_param all present . Change dtype to numeric where relevant
```
Comment (evacougnon) [inline aodndata/soop/soop_co2_nrt_nc_generator.py:229]: is that okay to remove the `raise InvalidFileContentError` if there are missing param?

File: test_aodndata/soop/test_soop_co2.py [hunk 2]
```diff
@@ -10,9 +10,7 @@
 GOOD_NC = os.path.join(TEST_ROOT, 'IMOS_SOOP-CO2_GST_20170126T023510Z_VNAA_FV01.nc')
 GOOD_FRMAP = os.path.join(TEST_ROOT, 'FutureReefMap_GST_20150518T124011Z_9V2768_FV01.nc')
 GOOD_ZIP = os.path.join(TEST_ROOT, 'IMOS_SOOP-CO2_GST_20170126T023510Z_VNAA_FV01.zip')
-GOOD_RT_IN_TXT = os.path.join(TEST_ROOT, 'IN_2018-022-0000dat.txt')
-GOOD_RT_IN_2_TXT = os.path.join(TEST_ROOT, 'IN_2019-345-0000dat.txt')
-GOOD_RT_AA_TXT = os.path.join(TEST_ROOT, 'AA_2019-032-0001dat.txt')
+GOOD_RT_IN_TXT = os.path.join(TEST_ROOT, 'IN_2022-332-0000dat.txt')
 
 ship_callsign_ls = {'VNAA': 'Aurora-Australis',
```
Comment (evacougnon) [inline test_aodndata/soop/test_soop_co2.py:15]: Do you still need to test files for the Aurora Australis?

**Review by evacougnon — COMMENTED** [review]
@bpasquer I went through it, I had a few minor comments mostly due to my lack of understanding on how this pipeline is meant to work

File: aodndata/soop/soop_co2_nrt_nc_generator.py (hunk 1 above)
Comment (bpasquer) [reply aodndata/soop/soop_co2_nrt_nc_generator.py:225]: I wanted to raise a warning instead but forgot. But you're right, best to keep it,i've  put the  `InvalidFileContentError` back in

File: test_aodndata/soop/test_soop_co2.py (hunk 2 above)
Comment (bpasquer) [reply test_aodndata/soop/test_soop_co2.py:15]: I need to keep testing  Aurora files in delayed mode only

**Comment by bpasquer** [comment]
I have followed you suggestion and added a unittest for missing params

File: aodndata/soop/soop_co2_nrt_nc_generator.py [hunk 3]
```diff
@@ -40,16 +42,11 @@
...
-                       'CO2um_m', 'H2Omm_m', 'DryBoxDruckPress', 'EquPress',
-                       'EquH2OFlow', 'LicorFlow', 'IntakeShipTemp', 'MetTrueWindSpKts',
-                       'MetTrueWindDir', 'AtmSeaLevelPress'}
-AA_SPECIFIC_INPUT_PARAMS = {'SBE45Flow', 'TsgSbe45Temp',
-                            'TsgSbe45Salinity'}
-IN_SPECIFIC_INPUT_PARAMS = {'TsgShipTemp',
-                            'TsgShipSalinity', 'TsgShipFlow', 'LabMainSwFlow'}
-
+                       'Li7815_CO2_avg', 'Li7815_H2O_avg', 'DryBoxDruckPress', 'EquPress',
+                       'EquH2OFlow', 'LicorFlow', 'Intake_T1','Intake_T2', 'MetTrueWindSpKts',
```
Comment (evacougnon) [inline aodndata/soop/soop_co2_nrt_nc_generator.py:46]: `Intake_T1` and `Intake_T2` still don't appear in the mapping, shall they be included? They're meant to be mapped with `TEMP_1_raw` and `TEMP_2_raw` further down

**Review by evacougnon — COMMENTED** [review]
just 2 variables you missed in the mapping described as the start of the soop_co2_nrt_nc_generator.py

**Review by evacougnon — APPROVED** [review]

**Pull request merged**